### PR TITLE
Jessiedlcs/update android build gradle

### PR DESCRIPTION
**Pull request by jessdelacruzsantos**
Updated reader-sdk version used

### CODE REVIEW

File: android/build.gradle [hunk 1]
```diff
@@ -28,7 +28,7 @@ buildscript {
 apply plugin: 'com.android.library'
 
 def DEFAULT_PLAY_SERVICES_BASE_VERSION = '16.0.1'
-def READER_SDK_VERSION = '[1.2.1, 2.0)'
+def READER_SDK_VERSION = '[1.3.0, 2.0)'
```
Comment (okenshields) [inline android/build.gradle]: should we upgrade to latest?

**Comment by okenshields** [comment]
do we need to bump the plugin version?

File: android/build.gradle (hunk 1 above)
Comment (hukid) [reply android/build.gradle]: I think we should since we've moved to AndroidX which is supported in 1.3.1 only.

Comment (jessdelacruzsantos) [reply android/build.gradle]: Copy that, bumping up to `1.3.1`

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "react-native-square-reader-sdk",
-  "version": "1.2.1",
+  "version": "1.2.2",
```
Comment (okenshields) [inline package.json]: Need to update CHANGELOG.md for this

**Pull request merged**